ios/RNMagicScript/components/UiNodes/UiListView/UiListViewNode.swift

```diff
@@ -164,9 +164,9 @@ import SceneKit
         setNeedsLayout()
     }
 
-    @objc override func hitTest(ray: Ray) -> BaseNode? {
+    override func hitTest(ray: Ray) -> HitTestResult? {
         guard let _ = selfHitTest(ray: ray) else { return nil }
-        return scrollView.hitTest(ray: ray) ?? self
+        return scrollView.hitTest(ray: ray)
```

Comment:
@pleszkiewicz shouldn't we rewrite this like:
```
  guard let result = selfHitTest(ray: ray) else { return nil }
  if let result = scrollView.hitTest(ray: ray) {
    return result
  }
  return result
```
?

Reply:
good point